pkgs/tools/misc/tmux-mem-cpu-load/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchgit, cmake }:
+
+stdenv.mkDerivation {
+  pname = "tmux-mem-cpu-load";
+  version = "v3.4.0";
+
+  src = fetchgit {
+    url = https://github.com/thewtex/tmux-mem-cpu-load.git;
+    rev = "54fdf3c68c13f7a5fd1a1e998d801d0a24dd910a";
+    sha256 = "1ybj513l4953jhayrzb47dlh4yv9bkvs0q1lfvky17v9fdkxgn2j";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "thewtex";
    repo = "tmux-mem-cpu-load";
    rev = "v${version}";
    sha256 = "1ybj513l4953jhayrzb47dlh4yv9bkvs0q1lfvky17v9fdkxgn2j";
  };
```

The hash probably changed. Please recalculate it.
Also unquoted URLs are deprecated.

---

Done